src/__tests__/Storyshots.test.tsx

```diff
@@ -1,10 +1,87 @@
 import initStoryshots from "@storybook/addon-storyshots"
+import { mount } from "enzyme"
+import * as React from "react"
+
+class ErrorBoundary extends React.Component<{ onError: (error) => void }> {
+  state = {
+    errorOccurred: false,
+  }
+
+  componentDidCatch(error, info) {
+    this.setState({ errorOccurred: true }, () => {
+      this.props.onError(error)
+    })
+  }
+
+  render() {
+    if (this.state.errorOccurred) {
+      return null
+    }
+    return this.props.children
+  }
+}
 
 initStoryshots({
   storyNameRegex: /Gene Follow/,
+  /**
+   * TODO:
+   *
+   * - API:
+   *   - Let story specify a sentinel selector to look for to exist.
+   *   - Let story specify a sentinel selector to look for to NOT exist.
+   *   - Maybe make it
+   *
+   * - Need to call tree.unmount() after run, but consider case where test times
+   *   out as well, so this should really run in `afterEach`. However, the
+   *   current option to specify before/after filters through storyshots is that
+   *   they are global, so no good way to scope e.g. `tree` to specific tests.
+   *
+   * - I saw some ‘snapshot filename’ support in storyshot, so we should respect
+   *   that and call `toMatchSnapshot` with the explicit filename.
```

Comment:
Moving all of these to your list of TODOs.

Reply:
👍  Yep we can definitely do this. It will feel much less jarring when developing